Parsers/ASimRegistryEvent/Parsers/ASimRegistry.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
Please rename this file to ASimRegistryEvent.yaml

Reply:
done